File: src/main/Java/boj_step2/boj14681.java
import java.io.*;

public class boj14681 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int x = Integer.parseInt(reader.readLine());
        int y = Integer.parseInt(reader.readLine());
        if (y > 0) {
            if (x > 0) writer.append(String.valueOf(1));
            else writer.append(String.valueOf(2));
        } else {
            if (x < 0) writer.append(String.valueOf(3));
            else writer.append(String.valueOf(4));
        }
        writer.close();
    }
}

File: src/main/kotlin/others/boj14681.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val x = r.readLine()!!.toInt()
    val y = r.readLine()!!.toInt()
    val result = if(x > 0 && y > 0) "1" else if(x < 0 && y > 0) "2" else if(x < 0 && y < 0) "3" else "4"
    w.write(result)
    w.close()
}